website/docs/r/hbase_instance.html.markdown

```diff
@@ -0,0 +1,88 @@
+---
+subcategory: "HBase"
+layout: "alicloud"
+page_title: "Alicloud: alicloud_hbase_instance"
+sidebar_current: "docs-alicloud-resource-hbase-instance"
+description: |-
+  Provides a HBase instance resource.
+---
+
+# alicloud\_hbase\_instance
+
+Provides a HBase instance resource supports replica set instances only. the HBase provides stable, reliable, and automatic scalable database services. 
+It offers a full range of database solutions, such as disaster recovery, backup, recovery, monitoring, and alarms.
+You can see detail product introduction [here](https://help.aliyun.com/product/49055.html?spm=5176.124785.766162.1.67273094TcUuke)
```

Comment:
Please use english docs link.
By the way, please remove the link suffix `?spm=xxxx`

Reply:
no english docs, next month supoort.